app/javascript/packs/micropost.js

```diff
@@ -0,0 +1,9 @@
+$(function(){
+    $('#micropost_image').bind('change', function() {
+      var size_in_megabytes = this.files[0].size/1024/1024;
+      if (size_in_megabytes > 5) { 
+        alert('Maximum file size is 5MB. Please choose a smaller file.')
+      }
+    });
+  });
+  
```

Comment:
```suggestion
  });

```
Remove last 2 spaces